sopel/plugin.py

```diff
@@ -1365,8 +1361,8 @@ def require_chanmsg(
 ) -> Callable:
     """Decorate a function to only be triggerable from a channel message.
 
-    :param str message: optional message said if triggered in private message
-    :param bool reply: use :meth:`~.bot.Sopel.reply` instead of
+    :param message: optional message said if triggered in private message
```

Comment:
This one is kinda interesting, since the annotation is _more_ misleading than the doc was, but I don't think it poses any actual problem for user understanding. Still an improvement to have one source of truth.

Reply:
Yeah, that `Union` is an unfortunate side effect of the established decorator dimorphism (usage supported both with and without `()`; can we call it that?).

But you didn't point out the real mistake in my work two lines below, forgetting to rewrap something 😛

Reply:
Oh, I see what it was about you said on IRC.

My recommendation here is to add a usage example in the docstring that shows that you can use it without parenthesis, as it is a valid syntax of Python and it is supported by this decorator.

Reply:
I've done examples for rather more than just this one: https://github.com/sopel-irc/sopel/pull/2539/commits/fd84b3e73ce2ab910e2a02572c573542e377804e

Enough to warrant re-reviews, I think. Sorry! But no force-push or rebase, so it should be easy to look at only the new changes. 😁